src/extension.ts

```diff
@@ -1286,6 +1286,63 @@ export async function activate(context: vscode.ExtensionContext) {
 		),
 	);
 
+	context.subscriptions.push(
+		vscode.commands.registerCommand(
+			'intuita.executeCodemodWithinPath',
+			async (uri: vscode.Uri) => {
+				try {
+					const { storageUri } = context;
+
+					if (!storageUri) {
+						throw new Error(
+							'No storage URI, aborting the command.',
+						);
+					}
+
+					const codemodList = await engineService.getCodemodList();
+
+					const codemodCodemodName =
+						await vscode.window.showQuickPick(
+							codemodList.map(({ name }) => name),
+							{
+								placeHolder:
```

Comment:
their API really has `placeHolder` instead of `placeholder`?

Reply:
<img width="322" alt="image" src="https://user-images.githubusercontent.com/125881252/234289355-d9d84d1d-c44e-4dbc-9b21-a63fc5a1d1c2.png">
Yes, they call it placeHolder 😄